app/labor/rate_limit_checker.rb

```diff
@@ -1,11 +1,16 @@
 class RateLimitChecker
   attr_reader :user, :action
 
+  def self.daily_account_follow_limit
+    ENV["RATE_LIMIT_FOLLOW_COUNT_DAILY"]&.to_i || 500
```

Comment:
this should use `ApplicationConfig["RATE_LIMIT_FOLLOW_COUNT_DAILY"]` and not `ENV` :)

Reply:
Hey @rhymes, I've changed this too. I haven't used this style of config before so I was a little lost. I did some looking up, is this from the [envied ](https://github.com/eval/envied) gem? It seems like a pretty awesome declarative way to manage this instead of just using environment variables.